style_guide.md

```diff
@@ -97,17 +97,14 @@ Assuming the rest of the guidance is followed, it is recommended to start the de
 - “Whether”, for a boolean field.
 - “The” with a plural noun phrase, for the description of an array of strings.
 
-  
-### 
-
 ## Documentation style guide 
 
 ### Bulleted lists
 When using bulleted lists you should:
 - start with a lead in line, followed by a colon
 - use lower case at the start of each bullet
-- do not put semicolons or full stops at the end of bullets
-- do not put or/and at the end of bullets
+- not put semicolons or full stops at the end of bullets
+- not put or/and at the end of bullets
 - try to have one sentence per bullet
 
 ### Restructured text files 
```

Comment:
I can't find any examples where properties have been marked up '\`\`propertyName\`\`' what is the significance of this? Can it be removed from the guide?